test/robot_test.coffee

```diff
@@ -462,6 +468,47 @@ describe 'Robot', ->
 
         expect(result).to.not.be.ok
 
+      it 'matches CatchAllMessages wrapping non-TextMessages', ->
```

Comment:
Random extra test case that I realized would be useful as I was writing tests for `catchAllAddressed`